### PR TITLE
Update vstest.md

### PR DESCRIPTION
**Pull request by ankit-arora12**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@ankit-arora12 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/test/vstest.md [hunk 1]
```diff
@@ -119,4 +119,9 @@ Data-driven tests that use xUnit and NUnit test frameworks have some known limit
 
 1. Test Impact Analysis 
 
-The above limitations are because of how the adapters for these test frameworks discover and report data-driven tests.
\ No newline at end of file
+The above limitations are because of how the adapters for these test frameworks discover and report data-driven tests.
+
+### Does the Vstest task support running tests targeting multiple target frameworks at a time?
+
+The Vstest task does not support running tests targeting multiple target frameworks at a time as this a limitation from [testplatform](https://github.com/microsoft/vstest/issues/2310) side.
```
Comment (ShreyasRmsft) [inline docs/pipelines/tasks/test/vstest.md:126]:
```suggestion
The VsTest task does not support running tests targeting multiple target frameworks at a time as this is a limitation from the [vstest platform](https://github.com/microsoft/vstest/issues/2310) side.
```

File: docs/pipelines/tasks/test/vstest.md [hunk 2]
```diff
@@ -119,4 +119,9 @@ Data-driven tests that use xUnit and NUnit test frameworks have some known limit
...
+If you want to run tests belonging to multiple target frameworks you'll need to have multiple instances of the vstest task one per set of dlls targeting a particular framework.
```
Comment (ShreyasRmsft) [inline docs/pipelines/tasks/test/vstest.md:127]:
```suggestion
If you want to run tests belonging to multiple target frameworks you'll need to have multiple instances of the vstest task, one per set of dlls targeting a particular framework.
```

**Comment by PRMerger10** [comment]
@steved0x : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by steved0x — APPROVED** [review]

**Comment by PRMerger14** [comment]
@steved0x : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by steved0x** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@steved0x: I'm sorry - only the author of this article, @shashban, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/pipelines/tasks/test/vstest.md [hunk 3]
```diff
@@ -119,4 +119,9 @@ Data-driven tests that use xUnit and NUnit test frameworks have some known limit
...
 1. Test Impact Analysis 
 
-The above limitations are because of how the adapters for these test frameworks discover and report data-driven tests.
\ No newline at end of file
+The above limitations are because of how the adapters for these test frameworks discover and report data-driven tests.
+
+### Does the Vstest task support running tests targeting multiple target frameworks at a time?
+
+The VsTest task does not support running tests targeting multiple target frameworks at a time as this is a limitation from the [vstest platform](https://github.com/microsoft/vstest/issues/2310) side.
+If you want to run tests belonging to multiple target frameworks you'll need to have multiple instances of the vstest task, one per set of dlls targeting a particular framework.
```
Comment (GitHubber17) [inline docs/pipelines/tasks/test/vstest.md:127]:
```suggestion
### Does the VsTest task support running tests that target multiple target frameworks at a time?

The VsTest task doesn't support running tests that target multiple target frameworks at a time as this is a limitation from the [vstest platform](https://github.com/microsoft/vstest/issues/2310) side.
If you want to run tests that belong to multiple target frameworks, you'll need multiple instances of the vstest task, one per set of dlls that target a particular framework.
```

**Comment by PRMerger10** [comment]
@GitHubber17 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**